src/ALE/regrid_edge_values.F90

```diff
@@ -27,7 +27,7 @@ module regrid_edge_values
                                           !! thickness for sum(h) in edge value inversions
 real, parameter :: hNeglect_dflt = 1.e-30 !< The default value for cut-off minimum
```

Comment:
No units on `hNeglect_edge_dflt` and `nNeglect_dflt`?  (guessing `[H]`)

Reply:
Good catch, but there is no way to give these variables units because you can't rescale the units of a parameter.  Instead, I plan to make the various optional `hNeglect` arguments mandatory (they are already present in every case), and use `hNeglect` as the default for `hNeglect_edge`, thereby eliminating the need for `hNeglect_dflt` and `hNeglect_edge_dflt` altogether.  However, I did not want to do that as a part of this PR, which only changes comments.

Reply:
Ok, but there are other parameters in this PR which have units, including the line directly below this one.

Reply:
Oh, maybe I misunderstood your comment.  I was referring to units in the comments, not actual rescaling of the value.

Reply:
I suppose the message is that only nondimensional numbers can be parameters, and these "parameters" are not long to live, so it does not need to be modified.